src/Core/src/Core/IPicker.cs

```diff
@@ -13,6 +13,11 @@ public interface IPicker : IView
 		/// </summary>
 		string Title { get; }
 
+		/// <summary>
+		/// Gets the color for the Picker title.
+		/// </summary>
+		Color TitleColor { get; }
```

Comment:
This should be a Brush.

Reply:
Totally agree. However, can we make it progressive? Start by merging this and other PRs related to Color, in order to complete Handlers and allow in Previews to work and port code in a simple way, and as we have to do with other important properties such as convert TextColor to Foreground, update little by little .

Reply:
Yeah, we can do that.